src/azure-cli/azure/cli/command_modules/backup/_help.py

```diff
@@ -350,6 +350,15 @@
     crafted: true
 """
 
+helps['backup recoverypoint move'] = """
+type: command
+short-summary: Move a particular recovery point of a backed up item from one tier to another tier.
+examples:
+  - name: Move a particular recovery point of a backed up item. (autogenerated)
+    text: az backup recoverypoint move --container-name MyContainer --item-name MyItem --resource-group MyResourceGroup --vault-name MyVault --name RpId --source-tier SourceTier --destination-tier DestinationTier
+    crafted: true
+"""
```

Comment:
If this is handwritten help instead of Aladdin, please remove the `autogenerated` tag
```suggestion
helps['backup recoverypoint move'] = """
type: command
short-summary: Move a particular recovery point of a backed up item from one tier to another tier.
examples:
  - name: Move a particular recovery point of a backed up item. 
    text: az backup recoverypoint move --container-name MyContainer --item-name MyItem --resource-group MyResourceGroup --vault-name MyVault --name RpId --source-tier SourceTier --destination-tier DestinationTier
"""
```

Reply:
Done